src/Mod/Arch/ArchAxis.py

```diff
@@ -1162,9 +1162,9 @@ def retranslateUi(self,widget=None):
         self.wLabel.setText(QtGui.QApplication.translate("Arch", "Total width", None))
         self.hLabel.setText(QtGui.QApplication.translate("Arch", "Total height", None))
         self.addRowButton.setText(QtGui.QApplication.translate("Arch", "Add row", None))
-        self.delRowButton.setText(QtGui.QApplication.translate("Arch", "Del row", None))
-        self.addColumnButton.setText(QtGui.QApplication.translate("Arch", "Add col", None))
-        self.delColumnButton.setText(QtGui.QApplication.translate("Arch", "Del col", None))
+        self.delRowButton.setText(QtGui.QApplication.translate("Arch", "Delete row", None))
+        self.addColumnButton.setText(QtGui.QApplication.translate("Arch", "Add column", None))
+        self.delColumnButton.setText(QtGui.QApplication.translate("Arch", "Delete column", None))
```

Comment:
These lines were written that way to make the dialog more compact... Extending them that way makes it occupy twice the width

Reply:
@yorikvanhavre reverted in 874d0e7